crates/storage/src/key.rs

```diff
@@ -346,7 +365,97 @@ macro_rules! impl_integer_key {
     }
 }
 
-impl_integer_key!(i8, u8, i16, u16, i32, u32, i64, u64, i128, u128);
+impl_integer_key!(
+    u8, u16, u32, u64, u128, i8, i16, i32, i64, i128, Uint64, Uint128, Uint256, Uint512,
+);
```

Comment:
Implement `Key` trait for `Uint{64,128,256,512}`.

Can't do the same for `Int`, `Udec`, `Dec` because they don't implement `Bytable`.